root/static/scripts/edit/URLCleanup.js

```diff
@@ -1,13 +1,29 @@
 /*
+ * @flow
  * Copyright (C) 2010 MetaBrainz Foundation
  *
  * This file is part of MusicBrainz, the open internet music database,
  * and is licensed under the GPL version 2, or (at your option) any
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
+import $ from 'jquery';
+import * as React from 'react';
```

Comment:
It's for the type annotation on https://github.com/metabrainz/musicbrainz-server/pull/2184/files#diff-a808bbcd17cb3e956e9ac0fce554d75f6e9db02e020bc03ca543c198ac8f8e06R4752, is there a better approach?

Reply:
I didn’t notice this piece of UI code at the bottom of this file implementing mostly logic otherwise. It should probably be split later on. Definitely not an issue of this PR.

@mwiencek: Would there be a way not to import these just for Flow definitions?

Reply:
Flow provides global references to the React types if you replace `.` with `$`, so `React$Node` is usable without importing anything.

Reply:
This has been fixed in commit <https://github.com/metabrainz/musicbrainz-server/pull/2184/commits/e85dfecb0e2bc39c67c4762accb4b6bb4b9359d3>.